java/core/src/test/org/apache/orc/impl/filter/TestPluginFilterService.java

```diff
@@ -56,4 +60,24 @@ public void testErrorCreatingFilter() {
   public void testMissingFilter() {
     assertTrue(FilterFactory.findPluginFilters("file://db/table11/file1", conf).isEmpty());
   }
+
+  @Test
+  public void testAllowListFilter() {
```

Comment:
Please add additional test cases explicitly and revise the logic.
- `testAllowListFilterAllowAll`
- `testAllowListFilterDisallowAll`